build-tools/packages/bundle-size-tools/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
```

Comment:
Can we use (extend) our shared config here?

Reply:
Unfortunately, extending our shared config here results in way too many changes needed. I took this .eslintrc specifically because it's a bit more lenient and there's precedent for doing it this way (see #10763)